### PR TITLE
make simple pino logging

### PR DESCRIPTION
**Pull request by raminLgh**

### CODE REVIEW

**Review by mkermani144 — CHANGES_REQUESTED** [review]
Please add `node_modules` to `.gitignore`.

**Review by mkermani144 — CHANGES_REQUESTED** [review]
Please relate the issue in the description. In addition, read the issue description carefully; it includes links to the tools that are needed for completing the task. Current log rotation implementation is done manually which is not what is not the best way to do it.

**Review by mkermani144 — CHANGES_REQUESTED** [review]
1. I get the following error when I run your code:
```
Error: 1h is neither a supported frequency or a number of milliseconds
```
2. Please put all these code in a log folder, so you can add code related to other tasks in the future.

**Review by mkermani144 — CHANGES_REQUESTED** [review]
I'm getting this error:
```
Error: No file name provided
```

**Comment by raminLgh** [comment]
are you sure ? I test it now and work for me
can you delete log file and run again ?

**Comment by mkermani144** [comment]
> are you sure ? I test it now and work for me can you delete log file and run again ?

Yes, I'm sure. I tried removing log file, too.

File: ramin/log/logService.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+});
+
+
+const logger = pino({
+  level: 'info',
+  transport: {
+    
+    targets: [
+      { target: 'pino-pretty', },
+      { target: 'pino/file', options: { destination: './logfile'}}
```
Comment (mkermani144) [inline ramin/log/logService.js:18]: We meant to use `pino-roll`. What happened to it?  
As you suggested in our meeting, `ping` function doesn't care about the passed `rollStream`, it just ignores it.

**Review by mkermani144 — CHANGES_REQUESTED** [review]

**Review by mkermani144 — APPROVED** [review]

**Pull request merged**